### PR TITLE
Refactored image endpoint, and image serializers.

### PR DESCRIPTION
**Pull request by jcairo**

### CODE REVIEW

File: author_api/serializers.py [hunk 1]
```diff
@@ -29,6 +26,19 @@ def to_internal_value(self, data):
         except:
             return None
 
+class PostImageSerializer(ImageSerializer):
```
Comment (derekdowling) [inline author_api/serializers.py:29]: Nice, looks good!

Comment (jcairo) [inline author_api/serializers.py:29]: Thanks for your input. Worked well.

**Comment by derekdowling** [comment]
Cool, what changes need to happen on the front end to support this?

**Comment by jcairo** [comment]
You guys are good. I've arranged it so you should be able to send posts in as json or multipart form if they have an image. Either way works. I'm working on author images and updating them now but should be same thing.

**Comment by derekdowling** [comment]
@KarimBaaba ^^

**Comment by xkb** [comment]
@derekdowling getting there...

**Comment by derekdowling** [comment]
I'm just letting you know that @jon made it nice and easy for you ;)

**Pull request merged**